core/camel-core/src/test/java/org/apache/camel/impl/engine/DefaultConfigurerResolverTest.java

```diff
@@ -0,0 +1,34 @@
+package org.apache.camel.impl.engine;
```

Comment:
Just need to add the ASF license header to this source code

Reply:
Yes, I noticed.
I launched `mvn clean verify -Psourcecheck` but forgot that the build does not fail if not adding `-Dcheckstyle.failOnViolation=true`.